workers/unity/Packages/com.improbable.gdk.playerlifecycle/Systems/PlayerHeartbeat/SendPlayerHeartbeatRequestSystem.cs

```diff
@@ -13,11 +13,10 @@ public class SendPlayerHeartbeatRequestSystem : ComponentSystem
         private struct Data
         {
             public readonly int Length;
-            [ReadOnly] public ComponentDataArray<PlayerHeartbeatClient.CommandSenders.PlayerHeartbeat> RequestSenders;
```

Comment:
why did you remove the `ReadOnly` tag?

Reply:
Because we alter the list inside this (even though under the hood this does not change the component)